src/lib/include/libsed.h

```diff
@@ -96,6 +96,13 @@ struct sed_opalv200_supported_feat {
 	uint8_t reserved2[5];
 } __attribute__((__packed__));
 
+struct sed_tper_properties {
+	struct {
+		char key_name[32];
```

Comment:
It would be great if you have defines for 32 and 23 magic numbers. Later you also use 23 in assert call. You should use define there.

Reply:
Makes sense. will do it